src/server/models/ActiveUserModel.ts

```diff
@@ -11,10 +12,10 @@ export interface ActiveUserSchemaDocument extends Document {
     id: string;
     email: string | null;
     loginId: string;
-    socketConnections: any[];
-    socketIds: any[];
-    currentChatId: string | null;
-    chatIds: any[];
+    socketConnections: Socket[];
+    socketIds: string[];
+    currentChat: string | null;
```

Comment:
Why use `currentChat` here, but `currentroom` in src/types/types.ts (ActiveUserType)?

Reply:
doesn't really affect it much, but i still changed it either way

Reply:
Ok. But I think it introduces inconsistency

Reply:
i agree, i was thinking too far on that one